Cargo.toml

```
@@ -53,9 +53,13 @@ lorawan = { package = "lorawan", path = "lorawan" }
 exponential-backoff = {git = "https://github.com/yoshuawuyts/exponential-backoff", branch = "master"}
 semtech-udp = { version = ">=0.9.4,<1", default-features=false, features=["server"] }
 helium-proto = { git = "https://github.com/helium/proto", branch="master", features=["services"]}
-helium-crypto = { git = "https://github.com/helium/helium-crypto-rs", tag = "v0.3.4", features = ["ecc608"]}
+helium-crypto = { git = "https://github.com/helium/helium-crypto-rs", tag = "v0.4.1", optional = true }
```

making helium-crypto optional causes a plain `cargo build` to fail
```suggestion
helium-crypto = { git = "https://github.com/helium/helium-crypto-rs", tag = "v0.4.1" }
```